vpc/main.tf

```diff
@@ -0,0 +1,83 @@
+provider "aws" {
```

Comment:
We need a `provider.tf` file for only our provider configuration file

Reply:
This will be created.